### DISCUSSION TITLE
Need Help

### DISCUSSION THREAD

**Discussion by BennieFun** [ehristoforu/DeFooocus#31] [Q&A]
.\python_embeded\python.exe: can't open file 'C:\\Users\\Documents\\DeFooocus_win64_portable\\DeFooocus\\entry_with_update.py': [Errno 2] No such file or directory
Press any key to continue . . .
downloaded and started the run, but this cam up on the first line? Have I missed anything to download?
I am running an AMD Ryzen 7 6800H with Radeon Graphics +16Gb RAM

**Comment by DEEVSAAN** [comment]
Is your problem solved, i was facing the same issue and then i resolved it
